Sources/PointFree/Environment.swift

```diff
@@ -50,29 +59,3 @@ public struct Environment {
     self.stripe = stripe
   }
 }
-
-public struct AppEnvironment {
```

Comment:
`AppEnvironment` and its static functions are replaced with `Current` and more general setter functions.